papas/simulation/Propagator.h

```diff
@@ -3,34 +3,35 @@
 #include "papas/detectors/Field.h"
 #include <memory>
 namespace papas {
-class PFParticle;
+
+class Particle;
 class Detector;
 class PFParticle;
 class SurfaceCylinder;
 
 class Propagator {
-/** Virtual class to be used to determine where a particle path crosses a detector cylinder
-*/
+  /** Virtual class to be used to determine where a particle path crosses a detector cylinder
+  */
 public:
   /** Constructor
    */
-  Propagator(std::shared_ptr<const Field> field): m_field(field) {};
+  Propagator(std::shared_ptr<const Field> field) : m_field(field){};
   /**
    Propagate particle to the selected cylinder and store the point where the particle crossed the cylinder
    @param[in] ptc particle that is to be propagated
    @param[in] cyl cylinder to which the particle is to be propagated.
    @param[in] field magnitude of magnetic field (used only for charged particles, defaults to zero if not set)
    */
-  virtual void propagateOne(const PFParticle& ptc, const SurfaceCylinder& cyl) const = 0;
-  /**
-   Propagate particle all cylinders of the detector
-   @param[in] ptc particle that is to be propagated
-   @param[in] detector  Detector through which to propagate
-   */
-  void propagate(const PFParticle& ptc, const Detector& detector) const;
+  virtual void propagateOne(Particle& ptc, const SurfaceCylinder& cyl) const = 0;
```

Comment:
Hi Alice, there should be a line break between methods, not sure it's the case here, and no linebreak between a docstring and the corresponding method. But no need to change anything else in this PR, you can always do that later.